### PR TITLE
removed tf.sigmoid ,used sigmoi_cross_entropy_with-logits is more simple

### PR DESCRIPTION
**Pull request by zhouxiaozhang**
removed tf.sigmoid ,used sigmoi_cross_entropy_with-logits is more simple

### CODE REVIEW

**Comment by aqibsaeed** [comment]
Thank you for looking into the code. I agree with your idea but the point of this implementation is to understand each component explicitly. Therefore at this point I am leaving pull request as is so others might look and incorporate this in their work, if they like.
